pkg/platforms/openstack/openstack.go

```diff
@@ -117,8 +100,12 @@ type OSPDeviceInfo struct {
 	NetworkID  string
 }
 
+func NewOpenstackContext() OpenstackInterface {
```

Comment:
we can't because we have multiple interfaces

Reply:
you can rename it to "New()" there is no name collision (openstack and openshift are different packages).

im fine either way or maybe just rename this one `NewOpenstack()`  since the "context" part doesnt bring much value to the method name.

up2u

Reply:
oh you are right sorry